airbyte-webapp/src/core/jsonSchema/schemaToYup.ts

```diff
@@ -68,6 +68,7 @@ export const buildYupFormForJsonSchema = (
       schema = yup.boolean();
       break;
     case "integer":
```

Comment:
Slightly tangential to this PR, but it looks like this code is not actually validating that the `"integer"` case is actually an integer. It seems to just treat it as a number, so non-integer values are actually allowed to be set (I just confirmed this by testing it out locally:
<img width="803" alt="Screen Shot 2022-12-05 at 11 38 48 AM" src="https://user-images.githubusercontent.com/22731524/205727482-061ace57-a9f6-4fe3-b3eb-641f917b719d.png">

So we should likely update that case to add a `yup.integer()` validation

Reply:
I added it to this related issue I created - there are a bunch of these low-hanging fruits: https://github.com/airbytehq/airbyte/issues/20064